.github/ISSUE_TEMPLATE/good_first_issue.md

```diff
@@ -0,0 +1,11 @@
+The purpose of the good first issue label is to highlight which issues are suitable for a new contributor without a deep understanding of the codebase.
+
+Useful skills:
+
+(For example “C++11 std::thread”, “Qt5 GUI and async GUI design” or “basic understanding of Bitcoin mining and the Bitcoin Core RPC interface”)
+
+Want to work on this issue?
+
+You do not need to request permission to start working on this. It is encouraged (but not compulsory) to comment on the issue if you are planning to work on it. This will help the maintainers of the project monitor which issues are actively being addressed and is also an effective way to request assistance if and when you need it.
+
+For guidance on opening a pull request please see [CONTRIBUTING.md](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md) 
```

Comment:
I think this can be stronger. I would almost like a disclaimer, like "[ ] Yes, I confirm I have read the contributer guidelines" ;)

```suggestion
For guidance on opening a pull request and other helpful tips, please read [CONTRIBUTING.md](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md) before starting the process.
```